tap-snapshots/test-arborist-load-actual.js-TAP.test.js

```diff
@@ -1967,8 +1967,7 @@ Node {
       "ghtgz": Object {
         "extraneous": true,
         "integrity": "sha512-yowslMd9y/lGBCDVO0RwZoXRK5X0zMsf6XECM6DdeqN7qwVnFQ6IAwJai7BD4mVe1xOdWWqWNkuzyuStvSBnHw==",
-        "resolved": "https://codeload.github.com/isaacs/abbrev-js/tar.gz/a9ee72ebc8fe3975f1b0c7aeb3a8f2a806a432eb",
-        "version": "npm:abbrev@1.1.1",
+        "version": "https://codeload.github.com/isaacs/abbrev-js/tar.gz/a9ee72ebc8fe3975f1b0c7aeb3a8f2a806a432eb",
```

Comment:
Interesting side effect of the changes in this: there was previously a bug where a remote tarball dep would be saved incorrectly as an alias dep.